tests/integration/raw_studies_blueprint/test_output_variables.py

```diff
@@ -150,3 +150,55 @@ def test_get_output_variables_imagrid_endpoint(client: TestClient, user_access_t
         ],
     }
     assert res.json() == expected_result
+
+
+def test_get_output_variables_view(client: TestClient, user_access_token: str, internal_study_id: str):
+    client.headers = {"Authorization": f"Bearer {user_access_token}"}
+    output_id = "20201014-1425eco-goodbye"
+    # Areas
+    body = {"type": "area", "variable_name": "OP. COST", "frequency": "weekly", "area_id": "de"}
+    res = client.get(f"/v1/studies/{internal_study_id}/output/{output_id}/variables-views/data", params=body)
+    assert res.json() == {"data": [[46452000.0, 46452000.0], [46452000.0, 46452000.0]], "columns": [1, 2]}
+    # Thermal clusters
+    body = {
```

Comment:
it's a little misleading to name it body, it will not be the body of the request but the query params